.docker/asciidoctor_2_0_10.patch

```diff
@@ -0,0 +1,14 @@
+Tracked at https://github.com/asciidoctor/asciidoctor/issues/3469
```

Comment:
I'm sure this sort of thing isn't normal but it seems nicer than trying to work around the bug in another way. Hopefully they'll just take the patch.